.gitignore

```diff
@@ -81,8 +81,10 @@ sorc/recentersigp.fd
 sorc/shave.fd
 sorc/wafs_awc_wafavn.fd
 sorc/wafs_blending.fd
+sorc/wafs_blending_0p25.fd
```

Comment:
Why is WAFS getting additional directories?

Reply:
In prior commit to develop (bringing in v16.0.7 changes from NCO) it included adding two missing sorc folder symlinks to link_fv3gfs.sh (which Yali Mao and SPA Jen noticed). They needed to be added to the .gitignore file but were overlooked in prior commit and I noticed in a fresh clone for this task. So the sorc folders already existed, just needed to ignore them.

Reply:
Definitely, we can open an issue with Yali for that. The specific sorc folder symlinks will most likely go away in the future anyway...SPA Jen indicated during chats about this that they don't see them as needed anymore so we can reevaluate them for v17.